app/perftools/management/commands/create_page_cache.py

```diff
@@ -49,6 +50,208 @@
 
 logger = logging.getLogger(__name__)
 
+
+def fetch_jtbd_hackathons():
+    db = JSONStore.objects.get(key='hackathons', view='hackathons')
+    status = db.data[0]
+    hackathons = db.data[1][0:2]
+    fields = ['logo', 'name', 'slug', 'summary', 'start_date', 'end_date', 'sponsor_profiles']
+    return [{k: v for k, v in event.items() if k in fields} for event in hackathons if status == 'upcoming' or status == 'current']
+
+
+def create_jtbd_earn_cache():
+    print('create_jtbd_earn_cache')
+    import datetime
+    from marketing.models import LeaderboardRank
+
+    top_earners = list(LeaderboardRank.objects.active().filter(
+        product='bounties', leaderboard='monthly_earners'
+    ).values('rank', 'amount', 'github_username').order_by('-amount')[0:4].cache())
+
+    thirty_days_ago = timezone.now() - datetime.timedelta(days=30)
+
+    bounties = list(Bounty.objects.filter(
+        network='mainnet', event=None, idx_status='open', created_on__gt=thirty_days_ago
+    ).order_by('-_val_usd_db').values('_val_usd_db', 'title', 'issue_description')[0:2])
+
+    # WalletConnect
+    featured_grant = Grant.objects.filter(pk=275).first()
```

Comment:
^ shouldn't this be grant with most matching as opposed to hardcoding walletconnect ?

Reply:
A friend of mine use to say "is what it is" :P.
But was asked to be that specific one.